cli/tests/stake.rs

```diff
@@ -347,6 +348,7 @@ fn test_stake_delegation_force() {
         lockup: Lockup::default(),
         amount: SpendAmount::Some(50_000_000_000),
         sign_only: false,
+        force: false,
```

Comment:
taking the test `test_stake_delegation_force_min_delegation` as an example, if I want to use this as a template for the min delegation check, I don't want the lack of votes on the vote account to interfere, how can I see a vote account that has a recent root slot for a test?

Reply:
Thanks trying that, the delegate will have little value with the force as it will still fail with a program error due to being below min delegation, but I've implemented to confirm it fails with and without force, on the create it'll be more useful. My local environment dies when I try run the test unfortunately, I've pushed it though so let's see how CI does

Reply:
I'm assuming there's a way to assert the specific error type but I don't know how, not sure how important this is to you? Just trying to get out for rust format hell right now to get ci to work

Reply:
looks like it passed now thx